### PR TITLE
fix: add imagePullSecrets to jobs

### PR DESCRIPTION
**Pull request by slushysnowman**
Add imagePullSecrets to post-install-jobs and pre-delete-jobs
Closes https://github.com/clastix/capsule/issues/486

### CODE REVIEW

File: charts/capsule/templates/post-install-job.yaml [hunk 1]
```diff
@@ -25,6 +25,10 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name | quote }}
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
+      {{- with .Values.imagePullSecrets }}
```
Comment (prometherion) [inline charts/capsule/templates/post-install-job.yaml:28]: Should we use the Manager pull secrets here, or maybe, would be better to provide different secrets just for this Job, such as `jobs.image.imagePullSecrets`?

Comment (slushysnowman) [reply charts/capsule/templates/post-install-job.yaml:28]: I'm doing a lot of stuff with charts at the moment - most places I see it just defined at global level for all images in the chart. Other charts like [bitnami/external-dns](https://github.com/bitnami/charts/blob/master/bitnami/external-dns/values.yaml) use a global value as well as individual values.

Either works for me to be honest - I'd be tempted to just use the main imagePullSecrets for simplicity's sake, and if a use case comes about where it's better to specify it separately for the jobs, then add it then, but if you think it's better to separate it from the get-go just let me know and I'll do that.

Comment (prometherion) [reply charts/capsule/templates/post-install-job.yaml:28]: No problem, it was just to start the discussion.

Let's implement this way, we have always room for improvement :)

**Review by prometherion — APPROVED** [review]
LGTM, if you want to get a patch release of this Chart, just bump up the version in the `Chart.yaml` file so we can tag it!

**Comment by slushysnowman** [comment]
@prometherion - Chart version bupmped and ready for merge! Thanks for the review!

**Pull request merged**